spec/integration/support/man_examples.rb

```diff
@@ -0,0 +1,27 @@
+#  Copyright (c) 2013-2015 SUSE LLC
+#
+#  This program is free software; you can redistribute it and/or
+#  modify it under the terms of version 3 of the GNU General Public License as
+#  published by the Free Software Foundation.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.   See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, contact SUSE LLC.
+#
+#  To contact SUSE about this file by physical or electronic mail,
+#  you may find current contact information at www.suse.com
+
+shared_examples "man" do
+  describe "man" do
+    it "calls the machinery man page" do
+      require 'byebug'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.